ctapipe/io/astropy_helpers.py

```diff
@@ -1,5 +1,6 @@
 #!/usr/bin/env python3
 """
+
```

Comment:
What are these new lines about? Are they from differences between the formatting produced by the current linters and the one you propose?

Reply:
I changed this file to show how the CI behaves when one submits a wrongly formatted file.  This should not be part of the PR.  I wanted to avoid opening a PR changing the CI, where everything works (could easily be false positives).